pyriemann/utils/distance.py

```diff
@@ -323,6 +323,65 @@ def distance_logeuclid(A, B, squared=False):
     return distance_euclid(logm(A), logm(B), squared=squared)
 
 
+def distance_poweuclid(A, B, p, squared=False):
+    r"""Power Euclidean distance between SPD/HPD matrices.
+
+    The power Euclidean distance of order :math:`p` between two SPD/HPD
+    matrices :math:`\mathbf{A}` and :math:`\mathbf{B}` is [1]_:
+
+    .. math::
+        d(\mathbf{A},\mathbf{B}) = \frac{1}{|p|}
+        \Vert \mathbf{A}^p - \mathbf{B}^p \Vert_F
+
+    Parameters
+    ----------
+    A : ndarray, shape (..., n, n)
+        First SPD/HPD matrices, at least 2D ndarray.
+    B : ndarray, shape (..., n, n)
+        Second SPD/HPD matrices, same dimensions as A.
+    p : float
+        Exponent. For p=0, it returns
+        :func:`pyriemann.utils.distance.distance_logeuclid`.
+    squared : bool, default False
+        Return squared distance.
+
+    Returns
+    -------
+    d : float or ndarray, shape (...,)
+        Power Euclidean distance between A and B.
+
+    Notes
+    -----
+    .. versionadded:: 0.7
+
+    See Also
+    --------
+    distance
+
+    References
+    ----------
+    .. [1] `Power Euclidean metrics for covariance matrices with application to
+        diffusion tensor imaging
+        <https://arxiv.org/abs/1009.3045>`_
+        I.L. Dryden, X. Pennec, & J.M. Peyrat. arXiv, 2010.
+    """
+    if not isinstance(p, (int, float)):
+        raise ValueError("Exponent p must be a scalar (Got {type(p)})")
```

Comment:
This is an f-string right? You forgot the f :)

Reply:
Yes, you are right!